### DISCUSSION TITLE
Several Questions

### DISCUSSION THREAD

**Discussion by DrSnowPengu** [GriefPrevention/GriefPrevention#848] [Q&A] [answered]
Hello RoboMVM,

I have several questions to ask you. 

1: What is your griefprevention discord link? I can't find it to join.

2: Are you still supporting and updating greifprevention because I was told by bloodshot that griefprevention was discontinued a while back or I must misunderstood it. I am currently using griefdefender and would love to switch back to griefprevention instead if you are still supporting it and continue to do so. If you are is there a way I can migrate the claims databases back to griefprevention?

3: I am also wondering on griefprevention how do I disable players placing chest to create a claiming area on the configuration? I can't remember how to on the configuration hahaha.

Sincerely,
Ace

**Comment by Jikoo** [comment, marked as answer]
1. No idea
2. GP for Bukkit has not stopped being maintained. Bloodshot used to maintain the port of GP for Sponge, which was very inconveniently also named GP. GP for Sponge was discontinued, not original GP for Bukkit.
3. Set `GriefPrevention.Claims.AutomaticNewPlayerClaimsRadius` to -1

**Comment by RoboMWM** [comment]
1. You've joined it before lol. Link is on same page as the IRC page.
1. Yes GP is alive and well, he means the sponge version, which he seems to like to omit to readily...
1. Jikoo has the answer.

**Comment by DrSnowPengu** [comment]
Hello, RoboMWM

1. I do not see the discord link provided in the IRS chat.
2. I'm really glad to hear that then and if that the case is there a way I can transfer all my GriefDefender claiming data's back to GriefPrevention, like migrating it?

**Comment by Muspah** [comment]
<Dumcord> <F​reeMonoid> Ace, if you come back: you can join Discord via https://discord.gg/qjQTg5K

**Comment by DrSnowPengu** [comment]
Ah I see, thank you! :)

**Comment by RoboMWM** [comment]
no idea if you can transfer the data back. The official link is http://r.robomwm.com/dumcord btw now that I recall. But the link is on that page it's just a bit below the chatbox.
